src/components/spreadsheet/spreadsheet.ts

```diff
@@ -293,8 +287,8 @@ export class Spreadsheet extends Component<SpreadsheetProps, SpreadsheetChildEnv
       loadCurrencies: this.model.config.external.loadCurrencies,
       loadLocales: this.model.config.external.loadLocales,
       isDashboard: () => this.model.getters.isDashboard(),
-      openSidePanel: this.openSidePanel.bind(this),
-      toggleSidePanel: this.toggleSidePanel.bind(this),
+      openSidePanel: this.sidePanel.open.bind(this.sidePanel),
+      toggleSidePanel: this.sidePanel.toggle.bind(this.sidePanel),
```

Comment:
I don't think those _need_ to be in the env now. The question is: do we want to keep them ? Bloating the env VS simpler syntax to open a side panel

Reply:
we should then think about  how we'ed like to open a side panel from an action?